gapis/service/service.proto

```diff
@@ -397,6 +397,9 @@ message ErrDataUnavailable {
   stringtable.Msg reason = 1;
   // If true, then making the same request at a later time may result in data.
   bool transient = 2;
+  // If true, draw call specific data was requested for a command range not
+  // containing any draw calls.
+  bool not_a_draw_call = 3;
```

Comment:
I made a comment about this on the original CL, but it got dropped with a force-push.
Why do you need this flag? The intention of `ErrDataUnavailable` is that you display `reason`. Why do you need to use another flag to display a different message?